dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java

```diff
@@ -248,6 +253,7 @@ public Page<CollectionRest> findSubmitAuthorizedByEntityType(
             List<Collection> collections = cs.findCollectionsWithSubmit(query, context, null, entityTypeLabel,
                                               Math.toIntExact(pageable.getOffset()),
                                               Math.toIntExact(pageable.getPageSize()));
+            Collections.sort(collections,  new NameAscendingComparator());
```

Comment:
@arvoConsultores : Rather than re-ordering results from Solr, wouldn't it be better performance to do this sort as part of the Solr query itself?  It looks like the `findCollectionsWithSubmit()` method just builds a Solr query, and we might just add `discoverQuery.setSortField()` call somewhere around here: https://github.com/DSpace/DSpace/blob/main/dspace-api/src/main/java/org/dspace/content/CollectionServiceImpl.java#L1015

Overall, I'd recommend we consider doing this sort using Solr **if it's possible to do so**.  That'd also provide much faster performance than reordering the Collection List after the results have been returned.  The other advantage to this approach is that all features which use `findCollectionsWithSubmit()` will then have the same alphabetical ordering with fewer code changes.

Reply:
Changed ordering to solr.